.github/workflows/buildImage.yml

```
@@ -38,6 +38,8 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          build-args: |
+            PY_VERSION=${{ matrix.python-version }}
           push: true
           tags: ${{ env.REGISTRY }}/${{ github.repository }}/learn-maintenance-image:latest${{ matrix.python-version }}
```

Let's put the python version in the image name and not the version tag:
```suggestion
          tags: ${{ env.REGISTRY }}/${{ github.repository }}/learn-maintenance-image-${{ matrix.python-version }}:latest
```